xbmc/cores/VideoRenderers/BaseRenderer.cpp

```diff
@@ -222,11 +222,14 @@ void CBaseRenderer::FindResolutionFromFpsMatch(float fps, float& weight)
 RESOLUTION CBaseRenderer::FindClosestResolution(float fps, float multiplier, RESOLUTION current, float& weight)
 {
   RESOLUTION_INFO curr = g_graphicsContext.GetResInfo(current);
-
+  RESOLUTION_INFO orig = curr;
```

Comment:
That's only a backup for the resolution user started with, before we tried to find a resolution. it's only used later on to find dimensions larger than it.